include/odp/api/spec/timer_types.h

```diff
@@ -202,10 +202,42 @@ typedef enum {
  *  This will be deprecated in the future. */
 #define ODP_CLOCK_EXT ODP_CLOCK_SRC_1
 
+/**
+ * Timer expiration mode
+ *
+ * Expiration mode selects how timer expiration time is interpreted. In the default mode
+ * (ODP_TIMER_EXP_DEFAULT), a timer implementation may round the requested expiration time up or
+ * down within resolution limits, and therefore application may receive timeouts slightly before or
+ * after the requested time. In ODP_TIMER_EXP_AFTER mode, timers are limited to expire exactly on
+ * the requested time or after it, but never before the time.
+ */
+typedef enum {
+	/** Expiration mode default
+	 *
+	 *  Timers may expire before or after the specified time. */
+	ODP_TIMER_EXP_DEFAULT = 0,
```

Comment:
Using "default" in the name might be confusing (e.g. interpreted as pointing to a higher level constant), maybe `ODP_TIMER_EXP_RELAXED`?

Reply:
Fixed in v2. Decided to change EXP_AFTER as the default value, since a validation test expected that already.